### ISSUE TITLE
Update onshore operators contact information

### ISSUE THREAD

**Issue by cthomasONRR** [ONRR/onrr.gov-site#1069]
https://www.onrr.gov/ReportPay/production-reporter-checklist.htm
![image.png](https://images.zenhubusercontent.com/60f9807ebc2bf779cc3921f2/6969c14a-5bbf-4055-a6db-5459d104e926)

This 303-231-3627 is now Alexis Long's phone number. We need to figure out the correct contact information to put online.

**Comment by cthomasONRR** [comment]
https://www.onrr.gov/reportpay/prod-checklist.htm
The production reporter checklist has Stacy and Dustin listed... maybe these are the correct contacts?
![image.png](https://images.zenhubusercontent.com/60f9807ebc2bf779cc3921f2/e89678d2-7b1b-44ea-a036-4865d5c6ca37)

**Comment by cthomasONRR** [comment]
OK super weird... there are 2 versions of this same New Production Reporter webpage:
This is the webpage when you navigate from the website: https://www.onrr.gov/reportpay/prod-checklist.htm
But if you use the search tool, you get this different/old one: https://www.onrr.gov/ReportPay/production-reporter-checklist.htm

**Comment by cthomasONRR** [comment]
I realized that this old/duplicate webpage (https://www.onrr.gov/ReportPay/production-reporter-checklist.htm) needed to be removed from the Y and Z drives. I removed them and Shannon set up a redirect so that people who bookmarked the old webpage are brought to https://www.onrr.gov/reportpay/prod-checklist.htm.

I will close this issue once the change is live and the redirect link is verified as working.
